ynh-build

```diff
@@ -82,6 +82,12 @@ function checkout_tag()
 
 function build()
 {
+   # check if the build branch type is coherent between what the user has specified and what is entered in the changelog
+   if ! [[ head -n 1 debian/changelog == *"$BRANCH"* ]]; then
```

Comment:
```suggestion
   if ! [[ "$(head -n 1 debian/changelog)" == *"$BRANCH"* ]]; then
```

Reply:
yeah i've done this on my tests but on the file x)